test/test_transforms_v2.py

```diff
@@ -847,8 +847,7 @@ def test_size_none(self, max_size, device, make_input):
         check_transform(
             transforms.Resize(size=None, max_size=max_size, antialias=True),
             make_input(self.INPUT_SIZE, device=device),
-            # atol=1 due to Resize v2 is using native uint8 interpolate path for bilinear and nearest modes
-            check_v1_compatibility=dict(rtol=0, atol=1),
+            check_v1_compatibility=False,
```

Comment:
Should we make this the same as the other `check_v1_compatibility` argument, where we keep the dictionary and only set it to False if size is None?

Reply:
This may not be needed here because `size` is always None in this test, and in any case we'll remove this test and instead use something like suggested in https://github.com/pytorch/vision/pull/8459/files#r1622182968

Reply:
Sounds good!